packages/matter-node.js/src/matter/cluster/server/CommandServer.ts

```diff
@@ -23,11 +23,11 @@ export class CommandServer<RequestT, ResponseT> {
         readonly name: string,
         protected readonly requestSchema: TlvSchema<RequestT>,
         protected readonly responseSchema: TlvSchema<ResponseT>,
-        protected readonly handler: (request: RequestT, session: Session<MatterDevice>, message: Message) => Promise<ResponseT> | ResponseT,
+        protected readonly handler: (request: RequestT | undefined, session: Session<MatterDevice>, message: Message) => Promise<ResponseT> | ResponseT,
```

Comment:
can the request really be undefined per the spec?

Reply:
The specs say that "if there are no parameters then the request can be omitted in the invoke structure". In fact here "RequestT" stands for "the parameters" ... but now that yousay it ... let me check something tomorrow